[2025-02-05T14:15:04.513+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-05T14:15:04.527+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: external_signal_workflow.task_1 manual__2025-02-05T14:15:02.902404+00:00 [queued]>
[2025-02-05T14:15:04.534+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: external_signal_workflow.task_1 manual__2025-02-05T14:15:02.902404+00:00 [queued]>
[2025-02-05T14:15:04.534+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-05T14:15:04.656+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonVirtualenvOperator): task_1> on 2025-02-05 14:15:02.902404+00:00
[2025-02-05T14:15:04.663+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1803) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-05T14:15:04.668+0000] {standard_task_runner.py:72} INFO - Started process 1804 to run task
[2025-02-05T14:15:04.664+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'external_signal_workflow', 'task_1', 'manual__2025-02-05T14:15:02.902404+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/external_signal_workflow.py', '--cfg-path', '/tmp/tmp6q3vw9em']
[2025-02-05T14:15:04.671+0000] {standard_task_runner.py:105} INFO - Job 69: Subtask task_1
[2025-02-05T14:15:04.718+0000] {task_command.py:467} INFO - Running <TaskInstance: external_signal_workflow.task_1 manual__2025-02-05T14:15:02.902404+00:00 [running]> on host ***-worker-3
[2025-02-05T14:15:04.788+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='external_signal_workflow' AIRFLOW_CTX_TASK_ID='task_1' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T14:15:02.902404+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T14:15:02.902404+00:00'
[2025-02-05T14:15:04.790+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-05T14:15:04.790+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-05T14:15:04.790+0000] {logging_mixin.py:190} INFO - Current task name:task_1 state:running start_date:2025-02-05 14:15:04.528373+00:00
[2025-02-05T14:15:04.791+0000] {logging_mixin.py:190} INFO - Dag name:external_signal_workflow and current dag run status:running
[2025-02-05T14:15:04.791+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-05T14:15:04.792+0000] {baseoperator.py:421} WARNING - PythonVirtualenvOperator.execute cannot be called outside TaskInstance!
[2025-02-05T14:15:04.793+0000] {process_utils.py:186} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvl0oz1fw7 --python=python
[2025-02-05T14:15:04.798+0000] {process_utils.py:190} INFO - Output:
[2025-02-05T14:15:05.662+0000] {process_utils.py:194} INFO - created virtual environment CPython3.12.8.final.0-64 in 367ms
[2025-02-05T14:15:05.663+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venvl0oz1fw7, clear=False, no_vcs_ignore=False, global=False)
[2025-02-05T14:15:05.664+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-02-05T14:15:05.664+0000] {process_utils.py:194} INFO -     added seed packages: pip==24.3.1
[2025-02-05T14:15:05.665+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-02-05T14:15:05.687+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvl0oz1fw7/bin/pip install -r /tmp/venvl0oz1fw7/requirements.txt
[2025-02-05T14:15:05.695+0000] {process_utils.py:190} INFO - Output:
[2025-02-05T14:15:07.041+0000] {process_utils.py:194} INFO - Collecting pandas==2.1.4 (from -r /tmp/venvl0oz1fw7/requirements.txt (line 1))
[2025-02-05T14:15:07.044+0000] {process_utils.py:194} INFO -   Using cached pandas-2.1.4-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (18 kB)
[2025-02-05T14:15:07.203+0000] {process_utils.py:194} INFO - Collecting numpy<2,>=1.26.0 (from pandas==2.1.4->-r /tmp/venvl0oz1fw7/requirements.txt (line 1))
[2025-02-05T14:15:07.206+0000] {process_utils.py:194} INFO -   Using cached numpy-1.26.4-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (62 kB)
[2025-02-05T14:15:07.255+0000] {process_utils.py:194} INFO - Collecting python-dateutil>=2.8.2 (from pandas==2.1.4->-r /tmp/venvl0oz1fw7/requirements.txt (line 1))
[2025-02-05T14:15:07.256+0000] {process_utils.py:194} INFO -   Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
[2025-02-05T14:15:07.315+0000] {process_utils.py:194} INFO - Collecting pytz>=2020.1 (from pandas==2.1.4->-r /tmp/venvl0oz1fw7/requirements.txt (line 1))
[2025-02-05T14:15:07.321+0000] {process_utils.py:194} INFO -   Using cached pytz-2025.1-py2.py3-none-any.whl.metadata (22 kB)
[2025-02-05T14:15:07.363+0000] {process_utils.py:194} INFO - Collecting tzdata>=2022.1 (from pandas==2.1.4->-r /tmp/venvl0oz1fw7/requirements.txt (line 1))
[2025-02-05T14:15:07.364+0000] {process_utils.py:194} INFO -   Using cached tzdata-2025.1-py2.py3-none-any.whl.metadata (1.4 kB)
[2025-02-05T14:15:07.403+0000] {process_utils.py:194} INFO - Collecting six>=1.5 (from python-dateutil>=2.8.2->pandas==2.1.4->-r /tmp/venvl0oz1fw7/requirements.txt (line 1))
[2025-02-05T14:15:07.405+0000] {process_utils.py:194} INFO -   Using cached six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
[2025-02-05T14:15:07.408+0000] {process_utils.py:194} INFO - Using cached pandas-2.1.4-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (14.3 MB)
[2025-02-05T14:15:07.438+0000] {process_utils.py:194} INFO - Using cached numpy-1.26.4-cp312-cp312-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (13.9 MB)
[2025-02-05T14:15:07.473+0000] {process_utils.py:194} INFO - Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
[2025-02-05T14:15:07.477+0000] {process_utils.py:194} INFO - Using cached pytz-2025.1-py2.py3-none-any.whl (507 kB)
[2025-02-05T14:15:07.481+0000] {process_utils.py:194} INFO - Using cached tzdata-2025.1-py2.py3-none-any.whl (346 kB)
[2025-02-05T14:15:07.484+0000] {process_utils.py:194} INFO - Using cached six-1.17.0-py2.py3-none-any.whl (11 kB)
[2025-02-05T14:15:07.549+0000] {process_utils.py:194} INFO - Installing collected packages: pytz, tzdata, six, numpy, python-dateutil, pandas
[2025-02-05T14:15:12.137+0000] {process_utils.py:194} INFO - Successfully installed numpy-1.26.4 pandas-2.1.4 python-dateutil-2.9.0.post0 pytz-2025.1 six-1.17.0 tzdata-2025.1
[2025-02-05T14:15:12.302+0000] {process_utils.py:194} INFO - 
[2025-02-05T14:15:12.303+0000] {process_utils.py:194} INFO - [notice] A new release of pip is available: 24.3.1 -> 25.0
[2025-02-05T14:15:12.303+0000] {process_utils.py:194} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-02-05T14:15:12.426+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvl0oz1fw7/bin/python /tmp/venv-calliikt9g0a/script.py /tmp/venv-calliikt9g0a/script.in /tmp/venv-calliikt9g0a/script.out /tmp/venv-calliikt9g0a/string_args.txt /tmp/venv-calliikt9g0a/termination.log
[2025-02-05T14:15:12.428+0000] {process_utils.py:190} INFO - Output:
[2025-02-05T14:15:13.083+0000] {python.py:240} INFO - Done. Returned value was: {"age":{"London":25.0,"New York":30.0,"Paris":35.0}}
[2025-02-05T14:15:13.145+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-05T14:15:13.146+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=external_signal_workflow, task_id=task_1, run_id=manual__2025-02-05T14:15:02.902404+00:00, execution_date=20250205T141502, start_date=20250205T141504, end_date=20250205T141513
[2025-02-05T14:15:13.159+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-05T14:15:13.160+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-05T14:15:13.160+0000] {logging_mixin.py:190} INFO - Dag name:external_signal_workflow queued_at:2025-02-05 14:15:02.944200+00:00
[2025-02-05T14:15:13.160+0000] {logging_mixin.py:190} INFO - Task hostname:***-worker-3 operator:PythonVirtualenvOperator
[2025-02-05T14:15:13.180+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-05T14:15:13.220+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T14:15:13.222+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
